traffic_ops/testing/api/v2/traffic_control_test.go

```diff
@@ -54,11 +54,5 @@ type TrafficControl struct {
 	SteeringTargets                      []tc.SteeringTargetNullable             `json:"steeringTargets"`
 	Serverchecks                         []tc.ServercheckRequestNullable         `json:"serverchecks"`
 	Users                                []tc.User                               `json:"users"`
-	Jobs                                 []JobRequest                            `json:"jobs"`
```

Comment:
I don't think you meant to include this change, right?

Reply:
Woops, you are correct.